source/extensions/filters/http/ratelimit/ratelimit.cc

```diff
@@ -159,6 +159,8 @@ void Filter::complete(Filters::Common::RateLimit::LimitStatus status,
     cluster_->statsScope().counterFromStatName(stat_names.ok_).inc();
     break;
   case Filters::Common::RateLimit::LimitStatus::Error:
+    const auto request_id_ = request_headers_->get(Http::Headers::get().RequestId)->value().getStringView();
+    ENVOY_LOG_TO_LOGGER(Logger::Registry::getLog(Logger::Id::filter), warn, "rate limit status,requestId={} status={}", request_id_, status);
```

Comment:
For debugging logs, `debug` is more appropriate than `warn`.
And we need to format this code by `tools/code_format/check_format.py fix`.